src/Api/Controllers/SendsController.cs

```diff
@@ -118,20 +151,40 @@ public async Task<SendResponseModel> PostFile()
 
             if (Request.ContentLength > 105906176) // 101 MB, give em' 1 extra MB for cushion
```

Comment:
I would add an additional check here, ` && !SelfHosted`, in other words, self-hosted folks should be able to upload whatever they want, 2GB, 1TB, whatever their server's local mapped storage can handle. The sysadmin of that self-hosted service can impose limits via their server or proxy configuration settings to limit upload size accordingly but we shouldn't impose this outside of our cloud environment for direct upload.